controllers/chat/group/leaveGroupMember.js

```diff
@@ -54,7 +54,7 @@ const leaveGroupMember = async (req, res) => {
     const {channel, channelApiResponse} = response.data || {};
 
     const {newChannelName, betterChannelMember, betterChannelMemberObject, updatedChannel} =
-      await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true);
+      await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `true` to `false` in the `updateBetterChannelMembers` function call could potentially affect the behavior of the system. If this function is expected to behave differently for group channels, it would be better to handle this logic inside the function itself rather than passing a flag from outside. This way, the function can encapsulate its own behavior and the calling code doesn't need to know about the internal workings of the function.

```diff
- await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, false);
+ await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse);
```

And then inside `updateBetterChannelMembers`, you can check the type of the channel:

```javascript
async function updateBetterChannelMembers(channel, channelApiResponse) {
  const isGroupChannel = checkIfGroupChannel(channel); // implement this function as needed
  if (isGroupChannel) {
    // handle group channel case
  } else {
    // handle other cases
  }
}
```
This approach improves the maintainability and readability of the code by keeping the logic related to how to update better channel members encapsulated within the function that performs the update.


<!-- This is an auto-generated comment by OSS CodeRabbit -->